assets/javascript/app.js

```diff
@@ -0,0 +1,199 @@
+$(document).ready(function(){
+  
+    // event listeners
+    $("#remaining-time").hide();
+    $("#start").on('click', trivia.startGame);
+    $(document).on('click' , '.option', trivia.guessChecker);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['trivia' was used before it was defined. (no-use-before-define)](https://app.codacy.com/app/columbia-fsf-f2019/trivia-game/pullRequest?prid=4555493)